### DISCUSSION TITLE
Please add support for `.frame(maxWidth: Double.infinity)`

### DISCUSSION THREAD

**Discussion by shial4** [skiptools/skip#33] [General]
I know frame `maxWidth` and `maxHeight` is low priority for you, however, I've been doing SwiftUI since the very beginning and have many apps build with it. 
I find this `frame` modifier very powerful and would love to have it fully supported by SKIP 
```
.frame(maxWidth: Double.infinity)
```

**Comment by marcprux** [comment]
Thanks. We plan on implementing this, and I've created an issue from this discussion at https://github.com/skiptools/skip-ui/issues/4

**Comment by aabewhite** [comment]
Fixed in skip-ui
